gam/clustering.py

```diff
@@ -538,8 +539,34 @@ def _update(self, count, mean, m2, new_values):
 
         return count, mean, m2
 
+    def _dask_update(self, count, mean, m2, new_values):
+        """Batch updates mu and sigma for bandit PAM using Welford's algorithm
+        Refs:
+            https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance
+            https://stackoverflow.com/questions/56402955/whats-the-formula-for-welfords-algorithm-for-variance-std-with-batch-updates
+
+        Args:
+            count (int): The number of reference points.
+            mean (int): The current mean
+            m2 (int): The updated mean
+            new_values (np.ndarray): The distance matrix
```

Comment:
should this be a da.array?

Reply:
since it's within the da.apply_along_axis it will be a numpy array

Reply:
ah gotcha. where is this being used?

Reply:
good point, I'm removing that and `_dask_finalize`